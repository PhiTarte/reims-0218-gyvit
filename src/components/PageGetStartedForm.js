import React from "react"
import { Container } from "reactstrap"
import GetStartedSignInEmail from "./GetStartedSignInEmail"
import GetStartedSignInTitle from "./GetStartedSignInTitle"
import GetStartedSignInButton from "./GetStartedSignInButton"
import GetStartedSignInLinkHaveAnAccount from "./GetStartedSignInLinkHaveAnAccount"

const PageGetStartedForm = () => (
  <Container
    style={{
      "padding-top": "25%",
      "margin-left": "30px",
      "background-color": "#ffffff",
      "margin-right": "50px"
    }}
  >
    <div>
      <GetStartedSignInTitle />
      <GetStartedSignInEmail />
      <GetStartedSignInButton />
      <GetStartedSignInLinkHaveAnAccount />
    </div>
  </Container>
)

export default PageGetStartedForm
