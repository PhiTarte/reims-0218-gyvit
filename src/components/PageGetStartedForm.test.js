import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import PageGetStartedForm from "./PageGetStartedForm"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<PageGetStartedForm />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<PageGetStartedForm />).toJSON()
  expect(tree).toMatchSnapshot()
})
