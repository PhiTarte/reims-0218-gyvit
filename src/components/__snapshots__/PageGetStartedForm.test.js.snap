// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
  style={
    Object {
      "background-color": "#ffffff",
      "margin-left": "30px",
      "margin-right": "50px",
      "padding-top": "25%",
    }
  }
>
  <div>
    <div>
      <p
        style={
          Object {
            "color": "#3b425d",
            "font-size": "24px",
            "font-stretch": "normal",
            "font-style": "normal",
            "font-weight": "normal",
            "height": "24px",
            "letter-spacing": "normal",
            "line-height": "normal",
            "text-align": "left",
          }
        }
      >
        Get started absolutely free
      </p>
      <p
        style={
          Object {
            "color": "#9da5c9",
            "fontSize": "18px",
            "fontStyle": "normal",
            "fontWeight": "normal",
            "height": "18px",
            "letter-spacing": "normal",
            "text-align": "left",
            "width": "300px",
          }
        }
      >
        Free forever. No credit card needed.
      </p>
    </div>
    <form
      className=""
      style={
        Object {
          "margin-bottom": "40px",
          "margin-top": "60px",
        }
      }
    >
      <div
        className="form-group"
      >
        <label
          className=""
          htmlFor="email"
          style={
            Object {
              "color": "#9da5c9",
              "fontSize": "12px",
              "fontStyle": "normal",
              "fontWeight": "normal",
              "height": "12px",
              "letter-spacing": "normal",
              "margin-top": "30px",
              "text-align": "left",
              "width": "33px",
            }
          }
        >
          Email
           
        </label>
        <input
          className="form-control"
          id="IdEmail"
          name="email"
          placeholder="Email"
          type="email"
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className=""
          htmlFor="email"
          style={
            Object {
              "color": "#9da5c9",
              "fontSize": "12px",
              "fontStyle": "normal",
              "fontWeight": "normal",
              "height": "12px",
              "letter-spacing": "normal",
              "margin-top": "30px",
              "text-align": "left",
              "width": "33px",
            }
          }
        >
          Name
           
        </label>
        <input
          className="form-control"
          id="IdEmail"
          name="name"
          placeholder="Name"
          type="email"
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className=""
          htmlFor="password"
          style={
            Object {
              "color": "#9da5c9",
              "fontSize": "12px",
              "fontStyle": "normal",
              "fontWeight": "normal",
              "height": "12px",
              "letter-spacing": "normal",
              "margin-top": "30px",
              "text-align": "left",
              "width": "75px",
            }
          }
        >
          Password
        </label>
        <input
          className="form-control"
          id="IdPassword"
          name="password"
          placeholder="Password"
          type="password"
        />
      </div>
    </form>
    <div>
      <button
        className="btn btn-info"
        onClick={[Function]}
        style={
          Object {
            "background-image": "linear-gradient(to left, #57aad1, #7a57d1)",
            "border-radius": "100px",
            "height": "54px",
            "width": "192px",
          }
        }
        type={undefined}
      >
        Continue
      </button>
    </div>
    <div
      style={
        Object {
          "color": "#3b425d",
          "fontSize": "16px",
          "fontStyle": "normal",
          "fontWeight": "normal",
          "height": "16px",
          "letter-spacing": "normal",
          "margin-top": "120px",
          "text-align": "left",
          "width": "300px",
        }
      }
    >
      <p>
        Already have an account ?
        <a
          className="nav-link"
          href="#"
          onClick={[Function]}
          style={
            Object {
              "display": "inline",
            }
          }
        >
          Sign in
        </a>
      </p>
    </div>
  </div>
</div>
`;
